package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@jupyterlab/latex",
-  "version": "3.1.0",
+  "version": "3.1.1",
```

Comment:
I would upgrade it to 3.2.0 to reflect the dependency upgrade as a patch version is expected to be a simple replacement.